### PR TITLE
steve/issue 7 ci

### PR DESCRIPTION
**Pull request by qwang98**
rustfmt and clippy check, with skip check option for markdown files and docs

rustfmt check taken from zkEVM

@leolara These edits change the source code (by fixing warnings and conforming to rustfmt). Most are cosmetic, but I'd still suggest you review them. Thanks!

### CODE REVIEW

File: examples/fibonacci.rs [hunk 1]
```diff
@@ -64,27 +64,25 @@ fn fibo_circuit<F: FieldExt>() -> chiquito::ir::Circuit<F, (), (u64, u64)> {
...
-                b = prev_a + b;
+                b += prev_a;
             }
 
             ctx.add(&fibo_last_step, (a, b));
         })
     });
 
-    //println!("{:#?}", fibo);
+    // println!("{:#?}", fibo);
```
Comment (ChihChengLiang) [inline examples/fibonacci.rs:79]:
```suggestion
```

**Review by ChihChengLiang — APPROVED** [review]
LGTM

**Comment by leolara** [comment]
@qwang98 please resolve conflicts and then merge

**Comment by qwang98** [comment]
some CI checks (formatting only) didn't pass, and I will fix all of them in the other issue

**Pull request merged**